enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/grpc/ExtAuthService.java

```diff
@@ -164,16 +164,19 @@ private CheckResponse buildResponse(CheckRequest request, ResponseObject respons
                 HeaderValueOption headerValueOption = HeaderValueOption.newBuilder()
                         .setHeader(HeaderValue.newBuilder().setKey(APIConstants.PATH_HEADER).setValue(constructedPath)
                                 .build()).build();
-                okResponseBuilder.addHeaders(headerValueOption);
+                    okResponseBuilder.addHeaders(headerValueOption);
             }
 
             if (responseObject.getHeaderMap() != null) {
                 responseObject.getHeaderMap().forEach((key, value) -> {
+                            if (("wso2_lambda_egress_gateway".equals(value)) && "x-wso2-cluster-header".equals(key)) {
```

Comment:
replace "x-wso2-cluster-header" with a constant. Add another constant for the other value too.

Reply:
Fixed by 9b38827